node-launchpad/src/components/status.rs

```diff
@@ -273,6 +274,11 @@ impl Status<'_> {
         Ok(())
     }
 
+    fn clear_node_items(&mut self) {
+        debug!("Cleaning items on Status page");
+        self.items.as_mut().unwrap().items.clear();
```

Comment:
```suggestion
        if let Some(items) = self.items.as_mut() {
        	items.clear();
        	debug!("Cleared the items on status page");
        }
```
Better to avoid unwraps..